lib/drivers/tandem/tandemDriver.js

```diff
@@ -1348,17 +1348,17 @@ module.exports = function (config) {
       carbSchedules[scheduleName] = carbSchedule;
       sensitivitySchedules[scheduleName] = sensitivitySchedule;
       targetSchedules[scheduleName] = targetSchedule;
-      insulinSettings[scheduleName] = {
-        duration : profile.insulin_duration,
-        units: 'minutes'
-      };
       bolusSettings[scheduleName] = {
         amountMaximum : {
           value: profile.max_bolus,
           units: 'Units'
         },
         calculator: {
-          enabled: profile.carb_entry ? true : false
+          enabled: profile.carb_entry ? true : false,
+          insulin: {
+            duration : profile.insulin_duration,
+            units: 'minutes'
```

Comment:
Shouldn't this be `hours`? The validation on the platform side is `0.0 <= x <= 10.0`